app/components/AuthRoutes.tsx

```diff
@@ -13,36 +11,29 @@ import { useBoolean } from '../utils/hooks';
 import { getTenants } from '../actions/tenants';
 import Auth from '../utils/auth';
 import Consts from '../utils/consts';
-import { getManagerData, getUserData, logout, receiveLogin } from '../actions/managers';
+import { getManagerData, getUserData, logout } from '../actions/managers';
 import Layout from '../containers/layout/Layout';
 import LicensePage from '../containers/LicensePage';
 import MaintenanceMode from '../containers/maintenance/MaintenanceModePageMessage';
 import SplashLoadingScreen from '../utils/SplashLoadingScreen';
 
-export default function AuthRoutes({ isSamlEnabled }) {
+const AuthRoutes: FunctionComponent = () => {
     const [isManagerDataFetched, setManagerDataFetched /* , unsetManagerDataFetched */] = useBoolean();
     const [isUserDataFetched, setUserDataFetched /* , unsetUserDataFetched */] = useBoolean();
```

Comment:
I think that there are some development leftovers 😁 

```suggestion
    const [isManagerDataFetched, setManagerDataFetched] = useBoolean();
    const [isUserDataFetched, setUserDataFetched] = useBoolean();
```

Reply:
Probably 🙂 Removed.